dubbo-common/src/main/java/org/apache/dubbo/common/constants/CommonConstants.java

```diff
@@ -546,4 +546,26 @@ public interface CommonConstants {
 
     String PREFER_JSON_FRAMEWORK_NAME = "dubbo.json-framework.prefer";
 
+    /**
+     * @since 3.1.0
+     */
+    String MESH_ENABLE = "mesh-enable";
+
+    /**
+     * @since 3.1.0
+     */
+    Integer DEFAULT_MESH_PORT = 80;
```

Comment:
Can we make the default mesh port the same as triple's default port? I think the default `80` port is not helpful for users using Triple, which is probably the most common case.